src/parenttext_pipeline/compile_flows.py

```diff
@@ -0,0 +1,41 @@
+from parenttext_pipeline.common import clear_or_create_folder
+from parenttext_pipeline import steps
+
+
+def run(config):
+    clear_or_create_folder(config.outputpath)
+    clear_or_create_folder(config.temppath)
+
+    input_file = None
+    input_file = "temp/parenttext_all_1_load_from_sheets.json"
+    for step_num, step_config in enumerate(config.steps):
+        output_file = apply_step(config, step_config, step_num + 1, input_file)
+        print(f"Applied step {step_config.type}, result stored at {output_file}")
+        input_file = output_file
+
+    steps.split_rapidpro_json(config, output_file)
+    print("Result written to output folder")
+
+
+STEP_MAPPING = {
```

Comment:
I use this pattern a few times, maybe there's a better way of doing this.

Reply:
If all the steps are in the `steps` module: `getattr(steps, name)`

Otherwise, it may be worth using function decorators.

```
STEP_MAPPING = dict()

def step(func):
    STEP_MAPPING[func.__name__] = func
    return func

def get_step(name):
    return STEP_MAPPING[name]

@step
def create_flows(...):
    ...
```